app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,135 @@
+// Place all the styles related to the landing controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+*{
+  margin: 0;
+  padding: 0;
+}
+
+@mixin linear-gradient($direction, $color-stops...) {
+  background: nth(nth($color-stops, 1), 1);
+  background: -webkit-linear-gradient(legacy-direction($direction), $color-stops);
+  background: linear-gradient($direction, $color-stops);
+}
+
+$orange: #f5c85f;
+$gray-bg: #EEEEEE;
+$dark-gray: #111111;
+$gray-text: #999999;
+
+$font-principal: 'Open Sans', sans-serif;
+
+$break-medium: 800px;
+
+html, body{
+  font: 100% $font-principal;
+  color: $gray-text;
+}
+
+.flex-list{
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: flex-start;
+}
+
+.decorator{
+  height:12px;
+  width: 100%;
+  background-image: url(asset_path("triangle.png"));
+}
+
+#principal{
```

Comment:
Don't use ID selectors to style CSS. ID selectors add too much specificity and prevents reusability